scipy/spatial/_spherical_voronoi.py

```diff
@@ -121,10 +120,25 @@ class SphericalVoronoi:
     ...                    c='g')
     >>> # indicate Voronoi regions (as Euclidean polygons)
     >>> for region in sv.regions:
-    ...    random_color = colors.rgb2hex(np.random.rand(3))
-    ...    polygon = Poly3DCollection([sv.vertices[region]], alpha=1.0)
-    ...    polygon.set_color(random_color)
-    ...    ax.add_collection3d(polygon)
+    ...    region_size = len(region)
+    ...    for i in range(region_size):
+    ...        if i == (region_size - 1):
+    ...            next_index = 0
+    ...        else:
+    ...            next_index = i + 1
+    ...        start = sv.vertices[region][i]
+    ...        end = sv.vertices[region][next_index]
+    ...        result = geometric_slerp(start, end, t_vals)
+    ...        ax.plot(result[...,0],
+    ...                result[...,1],
+    ...                result[...,2],
```

Comment:
It would be good if example code conforms to pep8/pycodestyle. Currently all of `_spherical_voronoi.py` conforms.

Reply:
ok, I added the spaces after the commas here